lib/ansible/modules/cloud/ovh/ovh_domain_zone_record.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovh_domain_zone_record
+short_description: Manage OVH domain zone records
+description:
+    - Manage OVH (French European hosting provider) domain zone records.
+version_added: "2.8"
+author: Loïc Latreille (@psykotox)
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh).
+      You have to create an application (a key and secret) with a consummer
+      key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
+requirements:
+    - ovh >  0.3.5
+options:
+    name:
```

Comment:
Should be "name" of the resource, no?
The resource is actually the dns zone record not the dns zone.

Reply:
The dns zone record can be empty, by example : if I want to modify the root MX records of a DNS zone.
So, the name in this context can not be the resource because the name can not be empty.

Reply:
yes, that makes sense